webpack.config.js

```diff
@@ -111,36 +145,10 @@ module.exports = {
       ignoreOrder: false, // Enable to remove warnings about conflicting order
       filename: './[name].min.css'
     }),
-    new RemovePlugin({
-      after: {
-        test: [
-          {
-            folder: 'assets/build/',
-            method: (filePath) => {
-              return [
-                'editorStyle.deps.json',
-                'style.deps.json',
-                'theme_antarctic.deps.json',
-                'theme_arctic.deps.json',
-                'theme_climate.deps.json',
-                'theme_forest.deps.json',
-                'theme_oceans.deps.json',
-                'theme_oil.deps.json',
-                'theme_plastic.deps.json'
-              ].filter(item => {
-                return new RegExp(item, 'm').test(filePath);
-              }).length > 0;
-            }
-          }
-        ]
-      }
-    }),
   ],
   optimization: {
-    ...defaultConfig.optimization,
```

Comment:
I checked the default config we're currently using, and it didn't have the `optimization` key, so this line didn't add anything. However when I check the latest version of Gutenberg, this file now does include it. https://github.com/WordPress/gutenberg/blob/trunk/packages/scripts/config/webpack.config.js#L108-L142

Reply:
Now it still seems better to not include it from the default config for now, otherwise it might result in surprises that could go unnoticed when we update to a later version of WordPress that includes this new base file. Probably a high chance that it doesn't play well with the config we already have.